.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+	"name": "Rust",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/rust:1-1-bookworm",
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				// cspell:disable-next-line
+				"foxundermoon.shell-format",
+				"streetsidesoftware.code-spell-checker"
```

Comment:
I noticed that this seems to try to load the cspell config from `/workspaces/coreutilsvscode/cspell.json`.
Adding a `./` to the setting in `.vscode/settings.json` to `{ "cSpell.import": ["./.vscode/cspell.json"] }` fixes this so that correctly loads `/workspaces/coreutils/.vscode/cspell.json`. I'm guessing this is bug in the extension perhaps?